source/jormungandr/tests/kirin_realtime_tests.py

```diff
@@ -2076,6 +2076,299 @@ def test_add_new_trip(self):
+        line_id = response['lines'][0]['id']
+        assert line_id == 'A'
+
+        response = self.query_region('commercial_modes?')
+        assert len(response['commercial_modes']) == 8
+        commercial_mode_id = response['commercial_modes'][0]['id']
+        assert commercial_mode_id == '0x0'
+
+        # Check that no departure exist on stop_point stop_point:stopC for neither base_schedule nor realtime
+        departure_query = "stop_points/stop_point:stopC/departures?_current_datetime=20120614T080000"
+        departures = self.query_region(departure_query + '&data_freshness=base_schedule')
+        assert len(departures['departures']) == 0
+        departures = self.query_region(departure_query + '&data_freshness=realtime')
+        assert len(departures['departures']) == 0
+
+        # Check stop_schedules on stop_point stop_point:stopC for base_schedule and realtime with
+        # Date_times list empty
+        ss_on_sp_query = "stop_points/stop_point:stopC/stop_schedules?_current_datetime=20120614T080000"
+        stop_schedules = self.query_region(ss_on_sp_query + '&data_freshness=realtime')
+        assert len(stop_schedules['stop_schedules']) == 1
+        assert stop_schedules['stop_schedules'][0]['links'][0]['type'] == 'line'
+        assert stop_schedules['stop_schedules'][0]['links'][0]['id'] == 'D'
+        assert len(stop_schedules['stop_schedules'][0]['date_times']) == 0
+
+        # Check that no stop_schedule exist on the line 'A' and stop_point 'stop_point:stopC'
+        ss_on_line_query = (
+            "stop_points/stop_point:stopC/lines/A/stop_schedules?_current_datetime=20120614T080000"
+        )
+        stop_schedules, status = self.query_region(ss_on_line_query + '&data_freshness=realtime', check=False)
+        assert status == 404
+        assert len(stop_schedules['stop_schedules']) == 0
+
+        # New disruption, a new trip with some attributes and with 2 stop_times in realtime
+        self.send_mock(
+            "additional-trip",
+            "20120614",
+            'added',
+            [
+                UpdatedStopTime(
+                    "stop_point:stopC",
+                    arrival_delay=0,
+                    departure_delay=0,
+                    is_added=True,
+                    arrival=tstamp("20120614T080100"),
+                    departure=tstamp("20120614T080100"),
+                    message='on time',
+                ),
+                UpdatedStopTime(
+                    "stop_point:stopB",
+                    arrival_delay=0,
+                    departure_delay=0,
+                    is_added=True,
+                    arrival=tstamp("20120614T080102"),
+                    departure=tstamp("20120614T080102"),
+                ),
+            ],
+            disruption_id='new_trip',
+            effect='additional_service',
+            physical_mode_id='physical_mode:Bus',  # this physical mode exists in kraken
+            headsign='trip_headsign',
```

Comment:
Can't you add dataset, too ?
`default:dataset` should be the uri of the one in dataset I think (to be tested before disruption).
Then to test that, the only way may be to request `vehicle_journeys/<vj_id>/datasets`.

Reply:
:+1: